pkg/virt-api/rest/subresource.go

```diff
@@ -787,6 +796,40 @@ func (app *SubresourceAPIApp) UnfreezeVMIRequestHandler(request *restful.Request
 
 }
 
+func (app *SubresourceAPIApp) ResetVMIRequestHandler(request *restful.Request, response *restful.Response) {
+
+	// The VMI must be assigned to a node for us to forward
+	// the request to virt-handler. Forward the
```

Comment:
nit: "Forward the", probably a left-over
```suggestion
	// the request to virt-handler.
```